### DISCUSSION TITLE
Getting Kafka-headers using the builtin Kafka connector

### DISCUSSION THREAD

**Discussion by morethom** [ConduitIO/conduit#1446] [Q&A] [answered]
Hi! 

Title says it all, I've been trying to migrate from Kafka-connect to Conduit for our replicators and find myself needing the Kafka headers from the messages coming from the source. I can't seem to find support for that and don't necessarily want to rewrite the kafka connector just for this. As anyone been able to get this data and/or is there plans for futur support. 

Thanks!

**Comment by maximus321** [comment]
finally someone said it, this is what has been preventing my migration.

**Comment by hariso** [comment]
Hey @morethom! Thanks for opening the discussion! I don't think there's any particular reason why we omitted the headers from the records. I'll open an issue for this. We'll get back to you soon with the planned milestone for it.

**Reply by hariso** [reply]
The mentioned issue: https://github.com/ConduitIO/conduit-connector-kafka/issues/125.

**Reply by morethom** [reply]
Thanks for the quick response @hariso ! This feature would make the Conduit migration alot easier for my team, love the project so far. I'll be waiting for a follow-up!

**Comment by hariso** [comment, marked as answer]
Here's the pull request: https://github.com/ConduitIO/conduit-connector-kafka/pull/126.
